requirements.txt

```diff
@@ -8,7 +8,7 @@ pytest==8.2.1
 python-decouple==3.8
 python-dotenv==1.0.1
 redis==5.0.4
-requests==2.31.0
+requests==2.32.2
```

Comment:
This version breaks Docker tests:
- https://github.com/docker/docker-py/pull/3257

Let's wait for the fix for `docker` package.